text/0030-standardized-sender-sig.md

```diff
@@ -31,7 +31,7 @@ to secrets involved in the construction of the TxOut.
 
 However, these numbers are useless for convincing a third-party who is not a party to the transaction,
 because only the recipient is able to validate these numbers. There is currently no form of receipt that
-can be validated just from the blockchain.
+can be validated by anyone, just from the blockchain.
```

Comment:
Is this an observation about the current state of things, or an indication that this MCIP solves this problem? My understanding is that this MCIP doesn't propose to have on-chain receipts that can be validated by anyone, but that the off-chain store could be, is that correct?